src/core/PlayAreaSelector.ttslua

```diff
@@ -70,8 +71,7 @@ function none() end
 
 -- main function (can be called by other objects)
 function updateSurface(newURL)
-    local playArea = getObjectFromGUID("721ba2")
-    local customInfo = playArea.getCustomObject()
+    local customInfo = playAreaAPI.getCustomObject()
```

Comment:
Instead of adding a single-use function to the Api, let's add the "updateSurface" functionality as an Api function